Parse/Internal/PFEncoder.m

```diff
@@ -188,8 +188,8 @@ - (id)encodeParseObject:(PFObject *)object {
 
 @interface PFOfflineObjectEncoder ()
 
-@property (nonatomic, assign) PFOfflineStore *store;
```

Comment:
There should be a static analysis rule for clang that allows for a warning for setting an object type to be assign. Its 2015 people!